tests/test_dse.py

```diff
@@ -635,7 +635,7 @@ def test_tti_rewrite_aggressive_opcounts(space_order, expected):
 
 @switchconfig(profiling='advanced')
 @pytest.mark.parametrize('space_order,expected', [
-    (4, 181), (12, 373)
+    (4, 315), (12, 779)
```

Comment:
we need to find out what's going on. This is clearly unacceptable without a proper explanation.